src/main/resources/swagger/api-docs.yaml

```diff
@@ -3638,7 +3638,51 @@ paths:
         414:
           description: |
             URI length exceeds the configured limit of 2048 characters.
-            Please use the POST endpoint when it's necessary to supply a large number of attribute names.
+            When necessary to supply a large number of attribute names, please use one of:
+            * POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv
+            * POST /cookie-authed/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/tsv
```

Comment:
A few tweaks to the summary and description of the existing GET tsv endpoint. No functional changes; just clarifying the doc.